src/components/header/header.style.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components/macro';
+import { Link } from 'react-router-dom';
+
+export const Container = styled.header`
+  height: 100px;
+  width: 100vw;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+export const Logo = styled.img`
+  margin-left: 80px;
+  width: 150px;
+  height: 36px;
```

Comment:
I don't think you need to set these because they are defined on the svg already

Reply:
You're right. Totally makes sense.